api/src/api/user.js

```diff
@@ -99,6 +99,24 @@ router.post(
   }),
 );
 
+// all user classes
+router.get(
+  '/classes', //changed
+  errorWrap(async (req, res) => {
+    const users = await User.find({}).select({
+      firstName: 1,
+      lastName: 1,
+      classes: 1,
+    });
+    res.status(200).json({
+      message: 'Successfully retrieved all users',
```

Comment:
Change this success message to describe what you retrieved (not just all users, you're getting all users' classes).